paper.md

```diff
@@ -0,0 +1,111 @@
+---
+title: 'Trash AI: A Web GUI for Serverless Computer Vision Analysis of Images of Trash'
+tags:
+  - tensorflow.js
+  - IndexDB
+  - Plastic Pollution
+  - Trash
+  - Litter
+  - AI
+  - Image Classification
+  - Serverless
+authors:
+  - name: Win Cowger
+    orcid: 0000-0001-9226-3104
+    affiliation: 1 # (Multiple affiliations must be quoted)
+  - name: Steven Hollingsworth
+    corresponding: true # (This is how to denote the corresponding author)
+    affiliation: 2
+  - name: Day Fey
+    affiliation: 2
+  - name: Mary C Norris
+    affiliation: 2
+  - name: Walter Yu
+    affiliation: 3
+  - name: Kristiina Kerge
+    affiliation: 4
+  - name: Kris Haamer
+    affiliation: 4
+  - name: Gina 
+    affiliation: 2
+  - name: Brianda
+    affiliation: 2
+affiliations:
+ - name: Moore Institute for Plastic Pollution Research, USA
+   index: 1
+ - name: Code for Sacramento and Fresno, USA
```

Comment:
Code for Sacramento and Open Fresno

Reply:
added https://github.com/code4sac/trash-ai/commit/4ff318f9c429d80adf701fee7131f2329e93818d